cli/src/action/mod.rs

```diff
@@ -36,6 +36,8 @@ use super::error::CliError;
 
 #[cfg(any(feature = "health", feature = "circuit"))]
 const DEFAULT_ENDPOINT: &str = "http://127.0.0.1:8085";
+#[cfg(any(feature = "health", feature = "circuit"))]
+const ENDPOINT_ENV: &str = "SPLINTERD_URL";
```

Comment:
Let's start this with SPLIINTER_ as a prefix (for all splinter env variables), so something like SPLINTER_DAEMON_URL,  SPLINTER_URL, etc.

Reply:
other options: SPLINTER_REST_API, SPLINTER_API_URL, SPLINTER_REST_URL

Reply:
Updated to SPLINTER_REST_API